{
    "docs": "https://docs.ansible.com/ansible-core/2.17/dev_guide/testing/sanity/validate-modules.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test validate-modules` [[explain](https://docs.ansible.com/ansible-core/2.17/dev_guide/testing/sanity/validate-modules.html)] failed with 14 errors:", 
            "output": "plugins/modules/customdns.py:0:0: doc-required-mismatch: Argument 'api_key' in argument_spec is required, but is not documented as being required\nplugins/modules/customdns.py:0:0: doc-required-mismatch: Argument 'url' in argument_spec is required, but is not documented as being required\nplugins/modules/customdns.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'pihole'\nplugins/modules/customdns.py:0:0: no-log-needed: Argument 'api_key' in argument_spec could be a secret, though doesn't have `no_log` set\nplugins/modules/customdns.py:0:0: nonexistent-parameter-documented: Argument 'name' is listed in DOCUMENTATION.options, but not accepted by the module argument_spec\nplugins/modules/customdns.py:0:0: nonexistent-parameter-documented: Argument 'new' is listed in DOCUMENTATION.options, but not accepted by the module argument_spec\nplugins/modules/customdns.py:0:0: parameter-type-not-in-doc: Argument 'api_key' in argument_spec defines type as 'str' but documentation doesn't define type\nplugins/modules/customdns.py:0:0: parameter-type-not-in-doc: Argument 'domain' in argument_spec defines type as 'str' but documentation doesn't define type\nplugins/modules/customdns.py:0:0: parameter-type-not-in-doc: Argument 'ip' in argument_spec defines type as 'str' but documentation doesn't define type\nplugins/modules/customdns.py:0:0: parameter-type-not-in-doc: Argument 'url' in argument_spec defines type as 'str' but documentation doesn't define type\nplugins/modules/customdns.py:0:0: undocumented-parameter: Argument 'api_key' is listed in the argument_spec, but not documented in the module documentation\nplugins/modules/customdns.py:0:0: undocumented-parameter: Argument 'domain' is listed in the argument_spec, but not documented in the module documentation\nplugins/modules/customdns.py:0:0: undocumented-parameter: Argument 'ip' is listed in the argument_spec, but not documented in the module documentation\nplugins/modules/customdns.py:0:0: undocumented-parameter: Argument 'url' is listed in the argument_spec, but not documented in the module documentation"
        }
    ], 
    "verified": false
}
